dev-packages/cli/src/util/git-util.ts

```diff
@@ -14,74 +14,159 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+import { resolve } from 'path';
 import * as sh from 'shelljs';
-import { fatalExec, getShellConfig } from './command-util';
-import { LOGGER } from './logger';
-import { validateGitDirectory } from './validation-util';
-
-export function isGithubCLIAuthenticated(): boolean {
-    LOGGER.debug('Verify that Github CLI is installed');
-    fatalExec('which gh', 'Github CLI is not installed!');
-
-    const status = sh.exec('gh auth status', getShellConfig());
-    if (status.code !== 0) {
-        if (status.stderr.includes('You are not logged into any GitHub hosts')) {
-            return false;
-        }
-        throw new Error(status.stderr);
-    }
-    if (!status.stderr.trim().includes('Logged in to github.com')) {
-        LOGGER.debug("No user is logged in for host 'github.com'");
-        return false;
-    }
-    LOGGER.debug('Github CLI is authenticated and ready to use');
-    return true;
+import { getShellConfig } from './command-util';
+
+export function isGitRepository(path?: string): boolean {
+    cdIfPresent(path);
+    return (
+        sh
```

Comment:
Odd formatting and unnecessary parenthesis, I believe. Was that formatted like this on purpose?

Reply:
Prettier autoformatting. Not really something I can change here 😉